File: JavaScript/loop/NStars.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
  });
  
  readline.question('', n => {
    // Write your code here
        // Print n stars horizontally
      let horizontalStars = '';
      for (let i = 0; i < n; i++) {
          horizontalStars += '* ';
      }
      console.log(horizontalStars.trim());
  
      // Print n stars vertically
      for (let i = 1; i < n; i++) {
          console.log('*');
      }
    // console.log(n)
    readline.close();
  });

File: Java/Loop/Nstars.java
package Java.Loop;

import java.util.Scanner;

public class Nstars {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Input
        int n = scanner.nextInt();

        // Print n stars horizontally
        for (int i = 0; i < n; i++) {
            System.out.print("* ");
        }
        System.out.println();

        // Print n stars vertically
        for (int i = 1; i < n; i++) {
            System.out.println("*");
        }

        scanner.close();
    }
}
